internal/usage/usage.go

```diff
@@ -26,11 +26,12 @@ const (
 	AuthDB                          = "Authentication database name."
 	Page                            = "Page number."
 	Status                          = "Alert's status."
+	Until                           = "Acknowledged until a date."
 	Limit                           = "Number of items per page."
 	Username                        = "Username for authenticating to MongoDB."
 	Password                        = "User’s password."
 	Roles                           = "User's roles and the databases or collections on which the roles apply."
-	Comment                         = "Optional description of the whitelist entry."
+	Comment                         = "Optional description of the whitelist entry or comment for the entry."
```

Comment:
This is still wrong, we can't keep `whitelist` there as is no longer true for alerts
```suggestion
	Comment                         = "Optional description or comment for the entry."
```